climada/hazard/tc_tracks.py

```diff
@@ -49,7 +49,7 @@
 
 # climada dependencies
 from climada.util import ureg
-import climada.util.coordinates as coord_util
+import climada.util.coordinates as co
```

Comment:
Since it also affected trop_cyclone.py, I used this PR to unify the shorthand for climada.util.coordinates everywhere in the whole project. The shorthand co is taken from the base modules in the hazard and exposures subpackages.